doc/howto/instances_create.md

```diff
@@ -343,7 +343,11 @@ You should now see the installer. After the installation is done, detach the cus
 Now the VM can be rebooted, and it will boot from disk.
 <!-- iso_vm_step7 end -->
 
-If the VM installed from an ISO is a Linux distribution using `systemd`, it is possible to install the LXD agent inside of it. This can be done manually or with the help of `cloud-init`:
+<!-- iso_vm_step8 start -->
+If the VM installed from an ISO is a Linux distribution using `systemd`, it is possible to install the LXD agent inside of it. This can be done manually or with the help of `cloud-init` (see [VM `cloud-init`](vm-cloud-init-config)).
```

Comment:
@simondeziel I was wondering if we should add this as an example to https://documentation.ubuntu.com/lxd/en/latest/cloud-init/#examples - but then I got confused where the `install.sh` comes from. Is that available somewhere if we use `cloud-init` without the `cloud-init` config device?

Reply:
@ru-fu that'd be a great addition to the cloud-init examples. Simply starting/creating a VM will have the proper config disk attached which always contains that `install.sh`.

Looking at https://documentation.ubuntu.com/lxd/en/latest/reference/devices_disk/#types-of-disk-devices (search for `cloud-init:config` as there are no more specific anchor), a config device is explicitly added because `cloud-init` on older releases like 18.04 doesn't look at the LXD datasource by default and thus needed an explicit config disk.

It took me a while to understand why 18.04's `cloud-init` needed an explicit config disk but this is due to how `cloud-init` backports are done: https://discourse.ubuntu.com/t/how-to-customize-lxd-image-metadata-for-cloud-init/25157